addons/website_sale/security/website_sale.xml

```diff
@@ -12,18 +12,6 @@
         <field name="perm_unlink" eval="False"/>
     </record>
 
-    <record id="product_supplierinfo_public" model="ir.rule">
-        <field name="name">Public product supplierinfo</field>
-        <field name="model_id" ref="product.model_product_supplierinfo"/>
-        <field name="domain_force">[('product_tmpl_id.website_published','=',True)]</field>
-        <field name="groups" eval="[(4, ref('base.group_public')), (4, ref('base.group_portal'))]"/>
-        <field name="perm_read" eval="True"/>
-        <field name="perm_write" eval="False"/>
-        <field name="perm_create" eval="False"/>
-        <field name="perm_unlink" eval="False"/>
-    </record>
```

Comment:
Ideally, this should be covered with a migration script removing the rule ;).
(because of the `noupdate="1"`).

Reply:
=> odoo/upgrade#910